jsonnet/vendor/github.com/stolostron/observatorium/jsonnet/lib/observatorium-api.libsonnet

```diff
@@ -25,6 +25,7 @@ local defaults = {
   tls: {},
   rateLimiter: {},
   internal: {},
+  securityContext: {},
```

Comment:
@jacobbaungard my update introduces this change (https://github.com/stolostron/observatorium/pull/79) is this going to be a problem?

Reply:
Hmmm, good question. I don't believe it would be a problem.

We do have this in the running obs-api container
```
          securityContext:
            privileged: false
            readOnlyRootFilesystem: true
```
It is added in the operator here:
https://github.com/stolostron/observatorium-operator/blob/12771fce4083e4cfbd9baaf573952e112115f5b3/jsonnet/obs-operator.jsonnet#L518

As far as I understand we add those settings on top of any set by the `securityContext` here. Might be worth to just double check that the deployment though when testing this change.